andela/the_spinners-frontend
# #17664422 solve app crashing when selecting accommodation

#### What does this PR do?
- Solve the bug that was causing the app to crash when no accommodation while selecting accommodations
- Add ability to get accommodations when they are available from backend
#### Description of Task to be completed?
#### How should this be manually tested?
Log in and try to select accommodation it should not fail to show you any data or some data(Try it using Heroku)
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#171664422](https://www.pivotaltracker.com/story/show/171664422)
#### Screenshots (if appropriate)
#### Questions:
